src/core/api/get_player_state.ts

```diff
@@ -74,6 +75,10 @@ export default function getLoadedContentState(
       return PLAYER_STATES.ENDED;
     }
 
+    if (stalledStatus.reason === "internal-seek") {
+      return isPlaying ? PLAYER_STATES.PLAYING : PLAYER_STATES.PAUSED;
```

Comment:
I forgot about that one.
Shouldn't we always return `PLAYER_STATES.BUFFERING` here? If we're stalled, why would we return `PLAYING` or `PAUSED`?

Reply:
To me, the point is just to not leak out seeking events when the seek operation happened in the RxPlayer's code.

Yet, if we are `stalled`, which means the content does not play, it's weird to lie about not being blocked. The UI won't be in sync here